server.js

```diff
@@ -19,14 +20,79 @@ mongoose.Promise = Promise;
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
+const listEndPoints = require('express-list-endpoints');
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+const { Schema } = mongoose;
+
+const quoteSchema = new Schema ({
+  sentence: String,
+  character: {
+  name: String,
+  slug: String,
+  house: {
+  name: String,
+  slug: String
+  }
+  }
+  })
+
+const Quote = mongoose.model("Quote", quoteSchema);
+
+if (process.env.RESET_DB) {
+  const resetDatabase = async () => {
+    await Quote.deleteMany();
+    topMusicData.forEach((singleQuote) => {
```

Comment:
Should not be called topMusicData I guess :)

Reply:
Haha maybe this is why it wasn’t really working for me this week 😅